bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/FileInfo.java

```diff
@@ -112,6 +114,29 @@ public long getSizeSinceLastwrite() {
         return sizeSinceLastwrite;
     }
 
+    public ByteBuffer getExplicitLac() {
```

Comment:
sure some folks might not use explicitLAC, but from the storage perspective, we should not have two LAC field in FileInfo. It would make code maintainance become much harder.